h/static/styles/annotation.scss

```diff
@@ -2,18 +2,52 @@
 
 //ANNOTATION CARD////////////////////////////////
 
+@mixin font-small {
+  font-size: $small-font-size;
+  line-height: $small-line-height;
+}
+
+@mixin font-normal {
+  font-size: $normal-font-size;
+  line-height: $normal-line-height;
+}
+
 // the padding at the left-edge of the annotation card
 // and its container
 $annotation-card-left-padding: 10px;
 
 .annotation-card {
-  background-color: white;
+  box-shadow: 0px 1px 1px 1px rgba(0, 0, 0, 0.10);
```

Comment:
This does not quite match the spec. The box-shadow in the non-hovered state is `0px 1px 1px 0px` (see https://trello-attachments.s3.amazonaws.com/571654d75b466478e53c0544/3500x600/68fc0bb587e3a1e79ca37230e8e7d279/Annotation_Card_-__specs.png ). Sounds very subtle but makes a difference to how much emphasis hovered cards have